<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>477</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 40pt &quot;WenQuanYi Zen Hei Sharp&quot;;
color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>公  交  查  询  系  统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelect">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>620</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;WenQuanYi Micro Hei&quot;;
background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 30pt &quot;WenQuanYi Zen Hei Sharp&quot;;
color: rgb(78, 154, 6);</string>
       </property>
       <property name="text">
        <string>起始地</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;WenQuanYi Micro Hei&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>830</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>760</y>
      <width>96</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(164, 0, 0);
font: 57 16pt &quot;WenQuanYi Zen Hei&quot;;</string>
    </property>
    <property name="text">
     <string>By Sheriby</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnManage">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>10</y>
      <width>224</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);
font: 57 16pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
    </property>
    <property name="text">
     <string>管理线路（管理员）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuery">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>60</y>
      <width>224</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);
font: 57 16pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
    </property>
    <property name="text">
     <string>查询公交线路</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>170</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 30pt &quot;WenQuanYi Zen Hei Sharp&quot;;
color: rgb(78, 154, 6);</string>
       </property>
       <property name="text">
        <string>目的地</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEnd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;WenQuanYi Micro Hei&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnJsonOut">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>120</y>
      <width>224</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);
font: 57 16pt &quot;WenQuanYi Zen Hei Sharp&quot;;</string>
    </property>
    <property name="text">
     <string>导出json数据</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionManage"/>
    <addaction name="actionQuery"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage">
   <property name="text">
    <string>管理线路</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>删除线路</string>
   </property>
  </action>
  <action name="actionQuery">
   <property name="text">
    <string>查询线路</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于本系统</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>寻求帮助</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>leStart</tabstop>
  <tabstop>leEnd</tabstop>
  <tabstop>btnSelect</tabstop>
  <tabstop>btnManage</tabstop>
  <tabstop>btnQuery</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
